### PR TITLE
Use absolute address to the log

### PR DESCRIPTION
**Pull request by scholzj**

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
 </script>
 <div align="center">
     <br />
-    <img alt="Strimzi" src="assets/strimzi.png" />
+    <img alt="Strimzi" src="https://strimzi.io/assets/strimzi.png" />
```
Comment (ppatierno) [inline _includes/header.html]: isn't it enough just `/assets/strimzi.png`?

Comment (scholzj) [reply _includes/header.html]: Might ... but maybe that will bubble out somwehere else. This sjhould be bulletproof :-)

Comment (ppatierno) [reply _includes/header.html]: Why not trying then? we have this cool preview feature thanks to netlify :-)

Comment (scholzj) [reply _includes/header.html]: I should have left you do it. We had the preview before as well ... and how much did it helped?

Comment (ppatierno) [reply _includes/header.html]: I forgot about it before :P

**Pull request merged**